src/librustc_yk_sections/emit_tir.rs

```diff
@@ -337,8 +362,48 @@ impl<'tcx> ToPack<ykpack::Place> for (&ConvCx<'_, 'tcx, '_>, &Place<'tcx>) {
         let (ccx, place) = self;
 
         match place {
-            Place::Base(PlaceBase::Local(local)) => ykpack::Place::Local(ccx.tir_var(*local)),
-            _ => ykpack::Place::Unimplemented, // FIXME
+            Place::Base(pb) => ykpack::Place::Base((*ccx, pb).to_pack()),
+            Place::Projection(pj) => ykpack::Place::Projection((*ccx, pj.as_ref()).to_pack()),
+        }
+    }
+}
+
+/// Projection -> Pack
+impl<'tcx, T> ToPack<ykpack::Projection<Box<ykpack::Place>, ykpack::LocalIndex>>
+    for (&ConvCx<'_, 'tcx, '_>, &Projection<'tcx, Place<'tcx>, Local, T>)
```

Comment:
Here we are taking a parameterised MIR `Projection` and from it making a parameterised TIR projection. I'd like to make the TIR projection concrete, especially since the MIR counterpart is only every constructed under one guise. The upstream PR suggests making the MIR struct concrete.

Reply:
OK.